core/enumerable/uniq_spec.rb

```diff
@@ -19,8 +19,16 @@ def each
         end
       end
 
-      it 'returns an array that contains only first argument of the yield' do
-        @enum.uniq { |_, label| label.downcase }.should == [0, 2]
+      ruby_version_is '2.5' do
+        it 'returns all yield arguments as an array' do
+          @enum.uniq { |_, label| label.downcase }.should == [[0, 'foo'], [2, 'bar']]
+        end
+      end
+
+      ruby_bug '#13669', ''...'2.5' do
```

Comment:
It's a typical synchronization problem.
Maybe I should stop testing against ruby-head or allow failure on Travis,
since MRI already runs ruby/spec on trunk.